js_modules/dagster-ui/packages/ui-core/src/instance/backfill/RunsFeedBackfillPage.tsx

```diff
@@ -0,0 +1,155 @@
+      return <NonIdealState icon="no-results" title={data.partitionBackfillOrError.message} />;
+    }
+
+    const backfill = data.partitionBackfillOrError;
+
+    return (
+      <Box style={{flex: 3, minHeight: 0}} flex={{direction: 'column'}}>
+        <Box padding={{left: 24}} border="bottom">
+          <Tabs size="large" selectedTabId={selectedTab}>
+            <Tab id="overview" title="Overview" onClick={() => setSelectedTab('overview')} />
+            <Tab id="runs" title="Sub-runs" onClick={() => setSelectedTab('runs')} />
+            <Tab id="timeline" title="Timeline" onClick={() => setSelectedTab('timeline')} />
+            {featureContext.canSeeBackfillCoordinatorLogs ? (
+              <Tab id="logs" title="Coordinator logs" onClick={() => setSelectedTab('logs')} />
+            ) : null}
+          </Tabs>
+        </Box>
+        {error?.graphQLErrors && (
+          <Alert intent="error" title={error.graphQLErrors.map((err) => err.message)} />
+        )}
+        <RunRequestContext.Provider
+          value={{buildLinkToRun: ({id}) => `/runs-feed/b/${backfillId}/${id}`}}
+        >
+          <Box flex={{direction: 'column'}} style={{flex: 1, position: 'relative', minHeight: 0}}>
+            {selectedTab === 'overview' && (
+              <Box style={{overflow: 'hidden'}} flex={{direction: 'column'}}>
+                {isDaemonHealthy ? null : (
+                  <Box padding={{horizontal: 24, top: 16}}>
+                    <DaemonNotRunningAlert />
+                  </Box>
+                )}
+
+                <BackfillOverviewDetails backfill={backfill} />
+                <BackfillAssetPartitionsTable backfill={backfill} />
+              </Box>
+            )}
+            {selectedTab === 'runs' && <BackfillRunsTab backfill={backfill} view="list" />}
+            {selectedTab === 'timeline' && <BackfillRunsTab backfill={backfill} view="timeline" />}
+            {selectedTab === 'logs' && <BackfillLogsTab backfill={backfill} />}
+          </Box>
+        </RunRequestContext.Provider>
+      </Box>
+    );
+  }
+
+  return (
+    <Box flex={{direction: 'column'}} style={{height: '100%', overflow: 'hidden'}}>
+      <PageHeader
+        title={
+          <Heading>
+            <Link to="/runs-feed" style={{color: Colors.textLight()}}>
+              All runs
+            </Link>
+            {' / '}
+            {backfillId}
+          </Heading>
+        }
+        right={
+          <Box flex={{gap: 12, alignItems: 'center'}}>
+            {isInProgress ? <QueryRefreshCountdown refreshState={refreshState} /> : null}
+            {backfill ? (
+              <BackfillActionsMenu
+                backfill={backfill}
+                refetch={queryResult.refetch}
+                canCancelRuns={backfill.status === BulkActionStatus.REQUESTED}
+              />
+            ) : null}
+          </Box>
+        }
+      />
+      {content()}
+    </Box>
+  );
+};
```

Comment:
This UI was more significantly different than the existing backfill detail page in earlier versions of the consolidated UI designs. 

It could make sense to fold this back into the old backfill page - right now it's just the URL, header on the page, and placement of the Overview info inside the overview tab instead of above the overview tab that are different.